package.json

```diff
@@ -17,11 +17,15 @@
     "env": "printenv",
     "clean:dist": "git checkout -- dist",
     "clean": "rm -rf generated dist lib .coverage && yarn run clean:dist",
-    "compile:node": "coffee --output lib --compile src",
-    "compile:test": "coffee --output generated/test --compile test",
+    "compile:node:coffee": "coffee --output lib --compile src",
```

Comment:
I'm guessing you figured this out because I see 7ae3f52682048c7fa7786d38c9a2679d797dc3cd but npm doesn't set the directories magically, either. Just a plain old $npm_package_* reference from package.json (has to exist or be part of the normalized package.json)